.cache/Jenkinsfile

```diff
@@ -3,17 +3,13 @@ pipeline {
     label "lead-toolchain-skaffold"
   }
   environment {
-    SKAFFOLD_DEFAULT_REPO = 'docker.artifactory.liatr.io/liatrio'
+    SKAFFOLD_DEFAULT_REPO = 'artifactory.toolchain.lead.prod.liatr.io/docker-registry/liatrio-dev'
```

Comment:
change to `liatrio` instead of `liatrio-dev`.

Also, need to update https://github.com/liatrio/springtrader/blob/solution1/skaffold.yaml#L10